src/Metadata/Put.php

```diff
@@ -92,8 +92,21 @@ public function __construct(
         ?string $name = null,
         $provider = null,
         $processor = null,
-        array $extraProperties = []
+        array $extraProperties = [],
+        private ?bool $allowCreate = null,
```

Comment:
IMHO the order of these parameters shouldn't be part of the BC promise and users **must** have to use named properties.

Reply:
note that this allows to use `...func_get_args()` to call `parent::__construct` without having to be concerned about `allowCreate`.